workspace/2019 DeepSpace/src/main/java/frc/robot/LifterHight.java

```diff
@@ -20,13 +20,13 @@
     private static final double maxHight = 196;
     private static final double minHight = 4;
 
-    private static final double ballFirstLevelHight = 67.945;
-    private static final double ballSecondLevelHight = 100;
-    private static final double ballThirdLevelHight = 150;
+    public static final double ballFirstLevelHight = 67.945;
```

Comment:
They need to be public.

The `Lifter` class will use these values in a bunch of comparison tests with `getDistance()` to determine how to move to get to various game piece levels. The code is really jumping around and is complicated, but if I had time I'd consolidate the two files into one. It would make more sense.